app/controllers/api/v1/restaurants_controller.rb

```diff
@@ -11,7 +11,8 @@ def show
     user = current_user
     render json: {
       restaurant: serialized_restaurant,
-      user: user
+      user: user,
+      votes: Vote.by_current_user(user)
     }
```

Comment:
We should be able to break this down to 
```
render json: Restaurant.find(params[:id])
```

Then we can pass along a user and votes in the serializer

Reply:
fixed this json, with just rendering the restaurant, and passing along the user and votes in the serializer